packages/client-gear-playground/webpack.config.babel.js

```diff
@@ -90,13 +90,13 @@ module.exports = {
           'css-loader?modules&localIdentName=[name]---[local]---[hash:base64:5]',
         ],
       },*/
-      {
-        test: /kununu-logo\/dist\/index.css$/,
-        use: [
-          'style-loader',
-          'css-loader?modules&localIdentName=[name]---[local]---[hash:base64:5]',
-        ],
-      },
+      // {
+      //   test: /kununu-logo\/dist\/index.css$/,
+      //   use: [
+      //     'style-loader',
+      //     'css-loader?modules&localIdentName=[name]---[local]---[hash:base64:5]',
+      //   ],
+      // },
```

Comment:
It's a little weird, that this is just commented out. Do you think we should remove it entirely?

Reply:
Well it's useful if you want to test the dist. I think once we redo this project it should be cleaned up.